src/pal/src/include/pal/virtual.h

```diff
@@ -105,6 +105,94 @@ BOOL VIRTUALOwnedRegion( IN UINT_PTR address );
 
 #ifdef __cplusplus
 }
+
+/// <summary>
```

Comment:
Sure. I can change the style of the comments. I assume you want this pattern:

/*++
…
--*/